README.md

```diff
@@ -1,31 +1,58 @@
-# Telegram app for Sailfish OS
-
-Depecher - telegram client for Sailfish OS based on [tdlib library](https://github.com/blacksailer/td/tree/cmake)
-
-
-Features:
-
-Depecher - Another Telegram client for Sailfish OS based on tdlib.
+# Depecher - A Telegram client for Sailfish OS
+based on tdlib.
 
 Features:
 - Send/View/Delete messages
 - View photos
 - View stickers
+- View gift
 - Uploading/Downloading photos/docs
 - Receive notifications
 - 2FA authorization enabled
+- and more...
+
+
+## Openrepos  
+
+
+https://openrepos.net/content/blacksailer/depecher
 
 
+## Contribute?
 
-#### Want to contribute?
 
 - Review code!
 - Create issue for missing features!
 - Do something really cool!
 
-To build this app required:
 
-1. tdlibjson tdlibjson-devel [rpm](https://openrepos.net/content/blacksailer/tdlibjson) package installed 
-2. APP_ID and APP_HASH changed in tdlibjson_wrapper.pro
+ ## Build 
+ 
+ 
+ - Installing dependencies
+
+Enter the sailfish os build machine (Start it from Tools/Options/Sailfish OS/Start Virtual Machine)
+
+ssh -p 2222 -i ~/SailfishOS/vmshare/ssh/private_keys/engine/mersdk mersdk@localhost
+
+- Enter the arch to use (use the arm)
+
+sb2 -m sdk-install -R -t SailfishOS-3.4.0.24-armv7hl
+    
+- Add the tdlibjson repo
+
+zypper ar http://repo.merproject.org/obs/home:/blacksailer:/branches:/home:/blacksailer/sailfish_latest_armv7hl/ tdlibsjon 
+    
+Refresh zypper repos
+
+zypper ref  
+
+- Install dependencies
```

Comment:
All dependencies are installed automatically so this is not really needed but you can keep it if you want.